setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name="yturl",
-    version="2.0.2",
+    version="2.0.3",
```

Comment:
We don't bump the version until the release tests pass, which are not run here. Please remove this. :-)